app/database.py

```diff
@@ -0,0 +1,30 @@
+from app import app
+from flask_sqlalchemy import SQLAlchemy
+from flask_wtf import FlaskForm
+from wtforms import StringField
+from wtforms.validators import DataRequired
+import os
+
+app.config["SECRET_KEY"] = os.getenv("SECRET_KEY")
```

Comment:
What do you think to move all getenv  to config.py file? There will be place to extract them and use default values if could be added. For example now you have always debug=True, but it should be managed by env

Here is example https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-xv-a-better-application-structure

You could have multiple configuration configs with default values. For production debug will be always False, for dev it could be True

Reply:
I think that's a brilliant idea! As suggested in the example, I've added config.py file and Config class.